src/openrct2/actions/LandSetHeightAction.hpp

```diff
@@ -207,7 +208,7 @@ DEFINE_GAME_ACTION(LandSetHeightAction, GAME_COMMAND_SET_LAND_HEIGHT, GameAction
             if (_height + 4 < tileElement->base_height)
                 continue;
             rct_scenery_entry* sceneryEntry = tileElement->AsSmallScenery()->GetEntry();
-            if (sceneryEntry->small_scenery.height > 64)
+            if (scenery_small_entry_has_flag(sceneryEntry, SMALL_SCENERY_FLAG_IS_TREE))
```

Comment:
Could you rename this function `CheckTreeObstructions`.

Reply:
Done.